int8 area
---
float32 bid